ffmpeg_tools/validation.py

```diff
@@ -277,15 +279,29 @@ def validate_unsupported_subtitle_streams(
     return True
 
 
-def validate_video_codec(video_format, video_codec):
-    if not formats.is_supported_video_codec(vformat=video_format, codec=video_codec):
-        raise UnsupportedVideoCodec(video_codec=video_codec, video_format=video_format)
+def validate_video_codecs(video_format, video_codecs):
+    assert formats.is_supported(video_format)
+
+    for video_codec in video_codecs:
+        if video_codec is None:
+            raise MissingVideoCodec
+
+        if not formats.is_supported_video_codec(vformat=video_format, codec=video_codec):
+            raise UnsupportedVideoCodec(video_codec=video_codec, video_format=video_format)
+
     return True
 
 
-def validate_audio_codec(video_format, audio_codec):
-    if not formats.is_supported_audio_codec(vformat=video_format, codec=audio_codec):
-        raise UnsupportedAudioCodec(audio_codec=audio_codec, video_format=video_format)
+def validate_audio_codecs(video_format, audio_codecs):
+    assert formats.is_supported(video_format)
+
+    for audio_codec in audio_codecs:
+        if audio_codec is None:
+            raise MissingAudioCodec
```

Comment:
Validation shouldn't fail in case of target codec. It should assume, that parameter doesn't change, if not specified. Related to review of https://github.com/golemfactory/golem/pull/4954#pullrequestreview-326881496